src/charts/line.js

```diff
@@ -750,13 +750,17 @@ define(function(require){
                 .selectAll('line')
                 .remove();
 
+            let minY = d3Array.min(dataByTopic, ({dates}) => d3Array.min(dates, getValue));
+            let shouldHighlight0Axis = minY < 0;
+            
             if (grid === 'horizontal' || grid === 'full') {
                 horizontalGridLines = svg.select('.grid-lines-group')
                     .selectAll('line.horizontal-grid-line')
                     .data(yScale.ticks(yTicks))
                     .enter()
                       .append('line')
-                        .attr('class', 'horizontal-grid-line')
+                        .classed('horizontal-grid-line', true)
```

Comment:
This is a bit of a nit, but I favor using attr('class', ...) when there is no logic involved on adding the class.